dydx/dydxViews/dydxViews/_v4/Receipt/dydxReceiptView.swift

```diff
@@ -43,13 +39,6 @@ public class dydxReceiptViewModel: PlatformViewModel {
             let children = self.children ?? []
             let items = self.collapsed ? Array(children.prefix(self.maxCollapsedItems - 1)) : children
 
-            let height: Double?
```

Comment:
I don't remember exactly why it's set to 132... maybe to accommodate the validation view.  Can you also test the UI with validation messages?

Reply:
https://github.com/dydxprotocol/v4-native-ios/assets/149746839/7f10411d-fbf7-4b09-97b7-b1a176912c9a